.github/workflows/win.yml

```
@@ -42,6 +42,7 @@ jobs:
         with:
           path: | # openssl and llvm take much longer to build so they are cached separately
             libs/pcre.lib
+            libs/pcre2.lib
```

```suggestion
            libs/pcre2-8.lib
```

(Also I didn't notice this wasn't cached before...)